### PR TITLE
Backend changes to enable a postgres deploy || Rachel Kolk

### PR DESCRIPTION
**Pull request by RachelKolk**
I also changed the titles and description in the reminders table to not be a unique field.

I had to rename tables so that the numbers go in order for the migration to run correctly. I've deployed it without errors on my own heroku deployment of our backend.
Once it deploys for us, if you pull the BE from master you'll have merge conflicts and need to accept my changes to the table naming to resolve them. :)

### CODE REVIEW

**Comment by RachelKolk** [comment]
This needs to be merged with the conflicts because the SQLLite database should not have been added to the repo and as a binary file there is no way to resolve the conflict.  I've added a commit that removes it from the repo and added it to the .gitignore.  

I'm not sure how to resolve this conflict at this point, someone with commit rights to master might have to remove that file.   Once we do that we can merge this and I can get postgres working on Heroku.

I setup another Heroku deployment off this branch on Heroku and tested it.  It works and you can check it out here - https://rk-reminders-be.herokuapp.com/api/users/

**Review by siratl — APPROVED** [review]

**Comment by cpoIT** [comment]
@josharmantrout91 @bummings How do we resolve the conflicting file:
database/reminders.db3?

**Review by cpoIT — APPROVED** [review]
@RachelKolk Is this resolved then?

**Pull request merged**
